project-overview.md

```diff
@@ -16,10 +16,12 @@ If the following resonates with any of your plans, if you would like to contribu
 ### Tokens
 
 Many real world attestation solutions and potential extrapolations of their use have been considered to ensure that the general logic within the components performs the correct set of operations to provide support for the maximum set of deployments. Developing understanding and modelling the necessary flows for their verification process has driven the Veraison design.
-To ensure that the code backs up the design, reference support for multiple attestation tokens will be developed in the project. The current targets for this support are:
+To ensure that the code backs up the design, reference support for multiple attestation tokens are supported in the project. The following are currently supported:
 
-* EAT Tokens, with the ARM PSA Profile being a specific test set.
-* DICE Tokens. NB: the current delivery format for the 'token', being the full DICE Certificate chain is still under consultation. If you're planning any such protocol implementations, we would be very interested in discussing them.
+* EAT Tokens, with the ARM PSA Profiles (Profile 1 & Profile 2) being a specific test set.
+* ARM Confidential Computing Architecture (CCA) Token and its variant [Parsec-CCA](https://github.com/CCC-Attestation/attested-tls-poc/blob/main/doc/parsec-evidence-cca.md)
+* [DICE Tokens](https://trustedcomputinggroup.org/work-groups/dice-architectures/) (note: this does not implement any specific DICE architecture). If you're planning any specific profile of DICE protocol implementations, we would be very interested in discussing them.
```

Comment:
```suggestion
* [DICE](https://trustedcomputinggroup.org/work-groups/dice-architectures/) based attestation tokens (note: this does not adhere to any specific DICE-based protocol). If you're planning any particular DICE implementations, we would be very interested in discussing them.
```

Should probably also remove the reference to "DICE Tokens" here -- DICE specifically doesn't define anything like a token format, so there no such thing as a "DICE Token". (just like there so no such thing as a "TPM Token").

For the same reason, the term "profile" isn't really applicable here.